compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GraphqlSourceModifier.java

```diff
@@ -470,6 +494,41 @@ private SpecificFieldNode getSchemaStringFieldNode(String schemaString) {
         return NodeFactory.createSpecificFieldNode(null, fieldName, colon, fieldValue);
     }
 
+    private SpecificFieldNode getFieldCacheConfigNode(CacheConfigContext cacheConfigContext) {
+        Node fieldName = NodeFactory.createIdentifierToken(FIELD_CACHE_CONFIG_FIELD);
```

Comment:
Are we checking whether the resource method has the `get` accessor? We should throw an error if the cache config is provided for a `subscribe` resource, right?